Tweak.x

```diff
@@ -1,10 +1,22 @@
 #import "../YouTubeHeader/YTVideoQualitySwitchOriginalController.h"
 
+BOOL makeItFake = NO;
+
+%hook YTVersionUtils
+
++ (NSString *)appVersion {
+    return makeItFake ? @"18.18.2" : %orig;
+}
+
+%end
+
 %hook YTVideoQualitySwitchControllerFactory
 
 - (id)videoQualitySwitchControllerWithParentResponder:(id)responder {
     Class originalClass = %c(YTVideoQualitySwitchOriginalController);
+    makeItFake = YES;
     return originalClass ? [[originalClass alloc] initWithParentResponder:responder] : %orig;
+    makeItFake = NO;
 }
 
-%end
\ No newline at end of file
+%end
```

Comment:
Why is there a change at EOL?

Reply:
It shouldn't be but i was editing it in the browser and may have accidentally deleted the line